symphony-bdk-http/symphony-bdk-http-jersey2/src/main/java/com/symphony/bdk/http/jersey2/ApiClientBuilderJersey2.java

```diff
@@ -240,6 +253,21 @@ protected SSLContext createSSLContext() {
       sslConfig
           .trustStoreBytes(trustStoreBytes)
           .trustStorePassword(trustStorePassword);
+      // if logging debug is enabled, we print the truststore entries
+      if (logger.isDebugEnabled()) {
+        final KeyStore truststore;
+        try {
+          truststore = KeyStore.getInstance(TRUSTSTORE_FORMAT);
+          truststore.load(new ByteArrayInputStream(trustStoreBytes), trustStorePassword.toCharArray());
+          final List<String> aliases = Collections.list(truststore.aliases());
+          logger.debug("Your custom truststore contains {} entries :", aliases.size());
+          for (String alias : aliases) {
+            logger.debug("# {}", alias);
```

Comment:
here we don't have much context on this log line and logs are often grepped/searched line by line
so I would probably just log for each alias something like:
`Loading {alias} from truststore'

and drop the line with the size

Reply:
I totally agree. However, It was done in Legacy in this way (see https://github.com/finos/symphony-bdk-java/pull/118/files#diff-5469b4312b60fd6917788efa41798433380dcc498ba75f7207c806da64b4f558R136)

Do we want to change logging to what you suggest?

Reply:
I would change it (my personal take) (given legacy and 2.0 are very different with very little compatibility)